### PR TITLE
65 feature request use application directories to store files

### PR DESCRIPTION
**Pull request by ntlhui**
Moves all log files, .ini files to appdir locations.  This also consolidates all RctGcs.config.Configuration references to the global singleton instance.

### CODE REVIEW

File: RctGcs/config.py [hunk 1]
```diff
@@ -9,6 +9,9 @@
 from socket import gaierror, gethostbyname
 from typing import Any, Dict, Tuple
 
+from appdirs import AppDirs
```
Comment (hgrehm) [inline RctGcs/config.py:12]: Imports not sorted

File: RctGcs/droneSimulator.py [hunk 2]
```diff
@@ -58,7 +58,7 @@
 import RCTComms.transport
 import utm
```
Comment (hgrehm) [inline RctGcs/droneSimulator.py:59]: Imports not sorted

File: RctGcs/ui/display.py [hunk 3]
```diff
@@ -3,20 +3,17 @@
 import logging
 import queue as q
 from functools import partial
-from pathlib import Path
-import RctGcs.config as config
-
-import RctGcs.rctCore
 
 import utm
```
Comment (hgrehm) [inline RctGcs/ui/display.py:7]: Imports not sorted

File: RctGcs/config.py (hunk 1 above)
Comment (ntlhui) [reply RctGcs/config.py:12]: This should be here per https://peps.python.org/pep-0008/#imports

File: RctGcs/droneSimulator.py (hunk 2 above)
Comment (ntlhui) [reply RctGcs/droneSimulator.py:59]: This should be here per https://peps.python.org/pep-0008/#imports

File: RctGcs/ui/display.py (hunk 3 above)
Comment (ntlhui) [reply RctGcs/ui/display.py:7]: These should be here per https://peps.python.org/pep-0008/#imports

**Review by hgrehm — APPROVED** [review]

**Pull request merged**